hashtable.py

```diff
@@ -1,14 +1,44 @@
+"""
+module hashtable
+"""
+
+
 class HashTable:
+    """
+    Implementation of hash table.
+    """
+
     def __init__(self):
-        pass
+        self.table = []
+        self.size = 10
+        for _ in range(self.size):
+            self.table.append({})
```

Comment:
Dictionaries are hash tables. So you use hashtable to implement hashtable

Reply:
resolved